packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -358,6 +354,18 @@ function toggleAllContainerGroups(value: boolean) {
                   </div>
                 </div>
               </td>
+              <td class="px-6 py-2 whitespace-nowrap w-10">
+                <div class="flex items-center">
+                  <div class="ml-2 text-sm text-gray-400"></div>
+                </div>
+              </td>
+              <td class="px-6 whitespace-nowrap">
+                <div class="flex flex-row justify-end opacity-0 group-hover:opacity-100">
+                  <!-- TODO: Figure out how to correctly pass in the id / engineid in a proper Typescript way. -->
+                  <PodActions
+                    pod="{{ id: containerGroup.id, name: containerGroup.name, engineId: containerGroup.engineId }}" />
```

Comment:
This part I need help on, I'm currently getting a warning in VSCode that I am not passing in every variable of PodInfoUI: https://github.com/containers/podman-desktop/blob/main/packages/renderer/src/lib/pod/PodInfoUI.ts#L25

Code wise, this works, but I'm unsure if this is okay in svelte / javascript?